server/src/main/java/com/cloud/configuration/Config.java

```diff
@@ -1415,6 +1447,8 @@ public enum Config {
     DefaultMaxDomainPublicIPs("Domain Defaults", ManagementServer.class, Long.class, "max.domain.public.ips", "40", "The default maximum number of public IPs that can be consumed by a domain", null),
     DefaultMaxDomainTemplates("Domain Defaults", ManagementServer.class, Long.class, "max.domain.templates", "40", "The default maximum number of templates that can be deployed for a domain", null),
     DefaultMaxDomainSnapshots("Domain Defaults", ManagementServer.class, Long.class, "max.domain.snapshots", "40", "The default maximum number of snapshots that can be created for a domain", null),
+    DefaultMaxDomainBackups("Domain Defaults", ManagementServer.class, Long.class, "max.domain.backups", "-1", "The default maximum number of backups that can be created for a domain", null),
+    DefaultMaxDomainBackupStorage("Domain Defaults", ManagementServer.class, Long.class, "max.domain.backup.storage", "-1", "The default maximum backup storage space (in GiB) that can be used for a domain", null),
```

Comment:
better keep some limit for domain backup & storage

Reply:
Done